torch/_inductor/scheduler.py

```diff
@@ -74,7 +75,9 @@
 fusion_log = torch._logging.getArtifactLogger(__name__, "fusion")
 loop_ordering_log = torch._logging.getArtifactLogger(__name__, "loop_ordering")
 
-PartitionType = list["BaseSchedulerNode"]
+PartitionType: TypeAlias = list["BaseSchedulerNode"]
+T = TypeVar("T")
```

Comment:
Does this have an __all__ export? If not, the typing extensions could be accidentally exported if they don't have an underscore.

Reply:
Done!  I'm going to fix the issue of `__all__` for good fairly soon.